src/core/main.c

```diff
@@ -875,6 +880,19 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case ARG_TEST:
                         arg_action = ACTION_TEST;
+
+                        if (optarg) {
+                                if (streq(optarg, "units"))
+                                        arg_dump_startup_seq |= DUMP_UNITS_SEQUENCE;
+                                else if (streq(optarg, "jobs"))
+                                        arg_dump_startup_seq |= DUMP_JOBS_SEQUENCE;
```

Comment:
This is a bit inconsistent with the man page, because the man page does not say anything about repeated arguments. Maybe use `=` not `|=`?